internal/providers/terraform/azure/notification_hubs.go

```diff
@@ -28,28 +28,34 @@ func NewAzureRMNotificationHubs(d *schema.ResourceData, u *schema.UsageData) *sc
 	costComponents = append(costComponents, notificationHubsCostComponent("Namespace usage", location, sku))
 	if u != nil && u.Get("monthly_pushes").Type != gjson.Null {
 		monthlyAdditionalPushes = decimalPtr(decimal.NewFromInt(u.Get("monthly_pushes").Int()))
-		if monthlyAdditionalPushes.GreaterThanOrEqual(decimal.NewFromInt(10000000)) {
-
-			if monthlyAdditionalPushes.GreaterThan(decimal.NewFromInt(10000000)) {
+	}
+	if sku != "Free" {
+		if sku == "Basic" {
+			if monthlyAdditionalPushes != nil {
+				pushLimits := []int{10000000}
+				pushQuantities := usage.CalculateTierBuckets(*monthlyAdditionalPushes, pushLimits)
+				if pushQuantities[1].GreaterThan(decimal.Zero) {
+					costComponents = append(costComponents, notificationHubsPushesCostComponent("Additional pushes (Over 10M)", location, sku, "10", &pushQuantities[1], 1000000))
```

Comment:
I'm wondering if we should remove the word "Additional" as the "over 10M" implies it for those pushes, likewise for the 100M. Up to you.